### ISSUE TITLE
use jshint to catch common mistakes

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#197]
In https://github.com/phetsims/chipper/issues/196#issuecomment-127263432, @samreid asked:

> Should we add a lint-ish rule that checks for the existence of the text "console.log" in certain chipper subdirectories?

Assuming that's possible with jshint...

That would be a nice enhancement, but not critical.  We might want to disallow "console.log" everywhere, since (presumably?) a built sim shouldn't be writing to the console.  Might also want to disallow "debugger".  Other things to disallow?

**Comment by samreid** [comment]
JSHint already fails for debugger statements.  I added a test one into hookes-law-main.js to demonstrate:

> Linting ../hookes-law/js/hookes-law-main.js ...ERROR
> [L23:C3] W087: Forgotten 'debugger' statement?

And our jshint style file has an entry for `console` as a read-only global.  We could remove that entry if we wish.

More generally, during the Bending Light code review I found that there were several places where it would be good to add jshint rules or augment with additional postprocessing to help automate some of the review-style work.
